extensions/ql-vscode/src/remote-queries/remote-queries-interface.ts

```diff
@@ -210,6 +211,9 @@ export class RemoteQueriesInterfaceManager {
       case 'remoteQueryDownloadAllAnalysesResults':
         await this.downloadAllAnalysesResults(msg);
         break;
+      case 'remoteQueryExportResults':
+        await await commands.executeCommand('codeQL.exportVariantAnalysisResults');
```

Comment:
Remove extra extra word.

```suggestion
        await commands.executeCommand('codeQL.exportVariantAnalysisResults');
```